src/styles/index.scss

```diff
@@ -1,12 +1,23 @@
-@import "bootstrap";
+@import './bootstrap';
+@import './variables';
```

Comment:
variables already included in bootstrap.
you can remove it from bootstrap.sass, but the order of imports should be changed